bitnami/external-dns/README.md

```diff
@@ -0,0 +1,124 @@
+# external-dns
+
+[ExternalDNS](https://github.com/kubernetes-incubator/external-dns) is a Kubernetes addon that configures public DNS servers with information about exposed Kubernetes services to make them discoverable.
+
+## TL;DR;
+
+```console
+$ helm install bitnami/external-dns
+```
+
+## Introduction
+
+This chart bootstraps a [ExternalDNS](https://github.com/bitnami/bitnami-docker-external-dns) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.4+ with Beta APIs enabled
+- PV provisioner support in the underlying infrastructure
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```console
+$ helm install --name my-release bitnami/external-dns
+```
+
+The command deploys ExternalDNS on the Kubernetes cluster in the default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation.
+
+> **Tip**: List all releases using `helm list`
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```console
+$ helm delete my-release
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Configuration
+
+The following table lists the configurable parameters of the external-dns chart and their default values.
```

Comment:
I do not know if it makes sense to point to a documentation page where the user can understand all the available values. For example, in the case of `policy` or `logLevel` I do not know which values I can set

Reply:
We can also provide the valid values. For instance: 

```
| Verbosity of the ExternalDN logs (accepted values: X, Y, Z,...)     |
```

 What do you think?

Reply:
Works for me